app/grandchallenge/api/urls.py

```diff
@@ -175,6 +175,7 @@
     basename="workstations-config",
 )
 router.register(r"workstations/sessions", SessionViewSet)
+router.register(r"workstations/feedback", FeedbackViewSet, basename="feedback")
```

Comment:
Nit: Could you put this in alphabetical order (above workstations/sessions)? The name should be `"workstations-feedback"` for consistency.